ptrace/debugger/child.py

```diff
@@ -143,6 +151,12 @@ def createChild(arguments, no_stdout, env=None):
      - env={} to start with an empty environment
      - env=None (default) to copy the environment
     """
+
+    if not RUNNING_WINDOWS:
+        if pass_fds and not close_fds:
+            warning("pass_fds overriding close_fds.")
+            close_fds = True
```

Comment:
I don't see why you must force close_fds=True here.

Reply:
I followed the python API for `subprocess.Popen`, where setting a value to `pass_fds` implies closing the other file descriptors.

From the official doc:
> pass_fds is an optional sequence of file descriptors to keep open between the parent and child. Providing any pass_fds forces close_fds to be True. (POSIX only)

Reply:
Ok. Please remove the warning.